### DISCUSSION TITLE
2011/02/28/%E8%B1%86%E8%B1%86%E7%9A%84%E5%93%AD%E5%92%8C%E7%AC%91

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#315] [Announcements]
# 2011/02/28/%E8%B1%86%E8%B1%86%E7%9A%84%E5%93%AD%E5%92%8C%E7%AC%91

豆豆两个半月大的时候，很多方面都比出生时有了较大变化，比如说哭吧：豆豆刚生出来的时候，哭起来嘴巴能张多大就张多大，声音极其响亮，哇哇哇的。这样的哭法与其说是为了引起别人同情，还不如说只是为了引起别人的注意。现在豆豆有了新哭法，如果不是特别着急的时候，他就会咧着嘴哭，声音也不大，哼哼唧唧的，再配合几滴眼泪，看起来就像是受了气，别提多委屈了。

https://blog.qizhen.xyz/2011/02/28/%E8%B1%86%E8%B1%86%E7%9A%84%E5%93%AD%E5%92%8C%E7%AC%91

**Comment by ruanqizhen** [comment]
之前博客下的留言：


勺子
2011年03月2日 22:22 编辑
初为人父，孩子的每一个微小的变化都是那么让人兴奋的，让人常常感叹生命的伟大。
勺最见不得女儿哭了，眼泪一流出来勺的心就软棉棉的了。

回复
ruanqizhen
2011年03月2日 08:49 编辑
豆豆是儿子:)

回复
Ted
2011年03月1日 21:49 编辑
幸福的爸爸和快乐的女儿

回复
